languages/scala/exercises/concept/basics/.docs/introduction.md

```diff
@@ -0,0 +1,105 @@
+## Defining Variables
+
+Defining a variable means assigning a value to a name. In Scala there are two kinds of variables, vals and vars.
+
+A `val` is [immutable][immutability], so it's value cannot be changed anymore after the first assignment:
+
+```scala
+val numberTen = 10
+
+numberTen = 11
+// --> raises a compile error "reassignment to val"
+```
+
+A `var`, in contrast, can be reassigned to a new value of same type:
+
+```scala
+var someNumber = 10
+
+someNumber = 11
+// now someNumber has been reassigned
+```
+
+Scala is a statically-typed language, which means that everything has a type at compile-time. A variable can be defined either by explicitly specifying its type, or by letting the Scala compiler infer its type based on the assigned value (known as [type inference][type-inference]). Therefore, the following two variable definitions are equivalent:
```

Comment:
In general, we try to not link to other resources from the `introduction.md` document. While this is a document that we wrote and thus is relatively basic, there is quite a bit of information in the linked document that the student has no need for (and will probably confuse the student), such as the implementing exercises and such.

```suggestion
Scala is a statically-typed language, which means that everything has a type at compile-time. A variable can be defined either by explicitly specifying its type, or by letting the Scala compiler infer its type based on the assigned value (known as type inference). Therefore, the following two variable definitions are equivalent:
```

Reply:
Makes sense, I removed them